source/renderer/app/components/hardware-wallet/HardwareWalletStatus.tsx

```diff
@@ -49,6 +49,12 @@ const messages = defineMessages({
     defaultMessage: '!!!Exporting the public key failed',
     description: '"Exporting public key failed" device state',
   },
+  unrecognized_wallet: {
```

Comment:
Should this be camel case?

Reply:
Here, I followed the existing pattern where we map the HwDeviceStatus enum values directly to translations. The enum uses snake_case therefore we have this mapping which doesn't follow JavaScript naming conventions. Would you like me to refactor this? I would have to introduce an additional map to map statuses to translation keys.

Reply:
Ah ok, sounds good.

Reply:
How about using [lodash's camelCase](https://lodash.com/docs/4.17.15#camelCase) for mapping enum values to translations? This way we could kill two birds with one stone. (btw it is a funny sentence)

Reply:
This would work, however I'm afraid that it will make it harder to refactor and track which translations are actually in use. Currently you can just find translations by a string, with camelCase function it's not going to be that easy anymore.